12-bookdown.Rmd

```diff
@@ -1,3 +1,630 @@
 # Books
 
-bookdown
+We have introduced the basics of R Markdown in Chapter \@ref(documents), which highlighted how HTML, PDF and Word documents can be produced from an R workflow.  The **bookdown** package [@R-bookdown] extends R Markdown and simplifies the creation of books and long-form documents. It allows for multiple R Markdown documents to easily be combined into a single book, adds additional features including numbering and cross-referencing figures/tables/sections/equations, inserting parts/appendices and can output elegant and appealing multi-page HTML book pages. This book itself is an example of how you can produce a book from a series of R Markdown documents, and both the printed version and the online version can look professional.
+
+Despite the package name containing the word "book", **bookdown** is not only for books. The "book" can be anything that consists of multiple R Markdown documents meant to be read in a linear sequence, such as course handouts, study notes, a software manual, a thesis, or even a diary. In fact, many **bookdown** features apply to single R Markdown documents as well (see Section \@ref(a-single-document)).
+
+In this chapter, we cover the basics of bookdown and explain how to get up and running with the project. Much of the the content is based on the work "bookdown: Authoring Books and Technical Documents with R Markdown" (https://bookdown.org/yihui/bookdown/) of @xie2016, which is recommended for reference more further explanation of the concepts.
+
+## Get started
+
+You can install either the CRAN version or the development version on GitHub (https://github.com/rstudio/bookdown):
+
+<!--- Is the convention of the book to explain that packages can be installed from GitHub? --->
+
+```{r eval=FALSE}
+# install from CRAN
+install.packages('bookdown')
+    
+# or GitHub
+devtools::install_github('rstudio/bookdown')
+```
+
+If you use RStudio, you can start a new bookdown project from the menu `File -> New Project... -> New Directory -> Book Project using bookdown`.^[Alternatively, the command `bookdown:::bookdown_skeleton(getwd())` (N.B. `:::`) will create a skeleton project in your current working directory.] Open the R Markdown file `index.Rmd` and click the button Build Book on the Build tab of RStudio. Now you should see the index page of this book demo in the RStudio Viewer, which looks like Figure \@ref(fig:bookdown-template).  
+
+You may add or change the R Markdown files, and hit the `Knit` button again to preview the book. If you prefer not to use RStudio, you may also compile the book through the command line. See Section \@ref(usage) for details.
+
+```{r bookdown-template, echo=FALSE, fig.cap= "The HTML output of the bookdown template", out.width='100%'}
+knitr::include_graphics("images/bookdown-template.png")
```

Comment:
Please add the argument `dpi = NA`.

If you have not optimized the image, I often use http://optimizilla.com to minimize the file size of images.